security/acme-client/src/opnsense/mvc/app/library/OPNsense/AcmeClient/LeValidation/DnsGoogleDomains.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
As already pointed out by others, this file needs to be renamed to `DnsGoogledomains.php`.

Reply:
@fraenki who would do that? I'm happy to help in whatever way I can :) Thanks!